docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -818,6 +818,47 @@ Default is the `permissions_config.toml` file in the [data directory](#data-path
 !!!note
     This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#permissions-configuration-file).
 
+### permissions-accounts-contract-address
+
+```bash tab="Syntax"
+--permissions-accounts-contract-address=<ContractAddress>
+```
+
+```bash tab="Command Line"
+--permissions-accounts-contract-address=xyz
+```
+
+```bash tab="Environment Variable"
+PANTHEON_PERMISSIONS_ACCOUNTS_CONTRACT_ADDRESS=xyz
+```
+
+```bash tab="Configuration File"
+permissions-accounts-contract-address=xyz
+```
+
+Specifies the contract address for [onchain accounts permissioning](../Permissions/Onchain-Permissioning/Onchain-Permissioning.md).
```

Comment:
```suggestion
Specifies the contract address for [onchain account permissioning](../Permissions/Onchain-Permissioning/Onchain-Permissioning.md).
```

Reply:
Not sure which way you want to go but further down you have [onchain node permissioning]

Reply:
Done